test/dummy/app/tasks/maintenance/params_task.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Maintenance
+  class ParamsTask < MaintenanceTasks::Task
+    attribute :post_ids, :string
+
+    validates :post_ids, presence: true
+    validates_format_of :post_ids, with: /\A([0-9]+(,\s?[0-9]+)*)\z/,
```

Comment:
We could use `\d` for `[0-9]`.
Why are spaces allowed after the comma but not before, and not at the end or beginning either? 😄 
-`1 ,2`-` 1,2`-`1,2 `-

Reply:
Yeah, we can make any of those valid 😂  I mostly just wanted it to work with submitting `1, 2, 3`